libs/designsystem/src/lib/scss/base/_variables.scss

```
@@ -110,3 +110,21 @@ $item-heights: (
 /* Dropdown
 ****************************************************************************/
 $dropdown-item-height: map-get($item-heights, 's');
+
+/* Transitions
+****************************************************************************/
+$transition-duration: (
+  quick: '200ms',
+  short: '300ms',
+  long: '500ms',
+  extra-long: '1000ms',
+);
+
+// Easing function
+$transition-easing: (
```

Align naming with existing maps:
```suggestion
$transition-easings: (
```

---

Done